SwedbankPaySDK/Classes/SwedbankPaySDK+Extensions/PaymentOrder.swift

```diff
@@ -325,8 +332,16 @@ public extension SwedbankPaySDK.PaymentOrderUrls {
         )
     }
     
+    /// Convenience initializer that generates a set of urls
+    /// for a payment using `MerchantBackendConfiguration`
+    ///  - parameter configuration: the MerchantBackendConfiguration where this payment is to be used
+    ///  - parameter language: the language of the payment
+    ///  - parameter hostUrl: the url to set in the hostUrls of the payment. This will also become the `webViewBaseURL` of the `ViewPaymentOrderInfo` created for this payment
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 174 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)